contracts/IAlluoStrategyV2.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Why is this file here? there is a copy of it in the `interface` folder ?

Reply:
Yes, this one can be removed indeed.

Reply:
Ok plz do, then I can approve